### PR TITLE
PENV-3: more observer configs

### PR DESCRIPTION
**Pull request by Vany**
add needed observer configs

### CODE REVIEW

File: run-test.py [hunk 1]
```diff
@@ -838,10 +838,12 @@ def deploy_observer(path, keep_database=False):
            "/../observer", flags="-r", ignore_errors=True)
     ssh(OBSERVER, "cd "+INSPATH +
         "/../observer && GO111MODULE=on make all && mkdir -p .artifacts")
-    scp_to(OBSERVER, "/tmp/insolar-jepsen-configs/observer.yaml",
-           INSPATH+"/../observer/.artifacts/observer.yaml")
-    scp_to(OBSERVER, "/tmp/insolar-jepsen-configs/observerapi.yaml",
-           INSPATH+"/../observer/.artifacts/observerapi.yaml")
+
+
+    for cqw in "observer observerapi stats-collector migrate".split(""):
```
Comment (afiskon) [inline run-test.py:843]:
```
>>> "observer observerapi stats-collector migrate".split("")
Traceback (most recent call last):
  File "<input>", line 1, in <module>
    "observer observerapi stats-collector migrate".split("")
ValueError: empty separator
```
@Vany really, please check your code before committing.

Comment (afiskon) [reply run-test.py:843]: Please consider just using regular lists.

Comment (Vany) [reply run-test.py:843]: oops

Comment (Vany) [reply run-test.py]: thx

**Pull request merged**